### ISSUE TITLE
Extend to Netherlands

### ISSUE THREAD

**Issue by mdehollander** [derjasper/TankenApp#3]
Hi,

Nice app and I think it would also be nice if we would have something like this for the Netherlands. I did not find a central database with API for the NL like tankerkoenig.de. But details of petrol stations with prices can be found here: http://www.anwb.nl/pois/5677ead2ed50d1a47a50dd14. If the output http://www.anwb.nl/pois/ could be limited to tankstation in a certain boundary this could serve as input. But I am not a API expert, so maybe you have an idea on how to do that.

Greetings.

**Comment by derjasper** [comment]
Thanks for your feedback! I will see what I can do. Related to #5 

Is this an open API and how is it licensed? Is there a documentation? I do not speak dutch and would be glad if you could help me ;)

**Comment by mdehollander** [comment]
The difficult part indeed is to find a good, documented, open API. The site is not an official api, I just noticed it when browsing their site. So there are no docs, license information etc. I also noticed that the link I send already expired. So it seems that here in the Netherlands there is not so much open data on petrol prices like in Germany. 
Even better would be to have a European or global database. It looks like Navx (http://www.opisnavx.com/) has this information, but again I do not see an open API.
Maybe you have any ideas how to search for an open API based on your experience? I am glad to help with some Dutch translations if needed.

**Comment by derjasper** [comment]
In the last days I extended the app so that the user can choose an api. I already added APIs for Austria (official data) and USA (user contributed data); it is pretty simple to add an api for Netherlands if there is a good api.

**Comment by mdehollander** [comment]
Nice! 
I did not find the perfect open API for the netherlands.
Maybe the site I send previous would be useful, although I am not sure if it is allowed to use it in this way. You can get a list of all petrol stations in json format at http://www.anwb.nl/pois/?types=tankstation. The coordinates are provided, although it is not possible to limit the results for a certain region/bbox/lat/lon. Would this be a problem? For each station details about prices can be obtained with another request.

Do you think it is possible to add this TankenApp or do we need a better API for the netherlands?

**Comment by derjasper** [comment]
I don't think that this api is sufficient since it would cause high server load. I would have to load one large file and download prices for each station. But besides that, I would not use a not public api without permission (except it's a governmental site).
